azure-pipelines-nightly.yml

```diff
@@ -205,49 +205,50 @@ steps:
      popd
     preferredAzurePowerShellVersion: 5.1.1
 
+
 - task: DotNetCoreCLI@2
   displayName: 'Spatial Test'
   inputs:
     command: 'test'
-    arguments: '--configuration $(BuildConfiguration) --no-build'

```

Comment:
why to run a certain collection, not the whole test project?

Reply:
collect code cov for each project